torchmultimodal/modules/encoders/image_encoder.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
rename to vision_transformer.py?

Reply:
`image_encoder.py` could be just a script for all types of image related encoders where `VisionTransformer` is just one of them? Rafi also has `text_encoder.py` with `BERTTextXXX`.